src/OneExplorer/OneExplorer.ts

```diff
@@ -229,6 +229,37 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<OneNode> {
         });
   }
 
+  delete(oneNode: OneNode): void {
+    if (oneNode.node.type === NodeType.directory) {
+      vscode.window
+          .showInformationMessage(
+              `Are you sure you want to delete '${
+                  path.parse(oneNode.node.path).base}' and its contents?`,
+              {detail: `You can restore this file from the Trash.`, modal: true}, 'Move to Trash')
+          .then(ans => {
+            if (ans === 'Move to Trash') {
+              Logger.debug('OneExplorer', `Delete '${path.parse(oneNode.node.path).base}'.`);
```

Comment:
How about chaingint this to `info`?
IMHO, deletion is important event and it would be better to have such log to track state when user encounters some issue with release vsix.